be/src/runtime/data_stream_mgr.cpp

```diff
@@ -111,16 +111,17 @@ Status DataStreamMgr::transmit_data(const PTransmitDataParams* request, ::google
         return Status::OK;
     }
 
+
```

Comment:
Please add comment for why you put this code here

Reply:
Ok, i will add some comments for the reason.